.browserslistrc

```diff
@@ -0,0 +1,4 @@
+chrome >= 79
+firefox >= 85
+safari >= 13
+edge >= 88
```

Comment:
Only chrome needed a drop in supported version?

Reply:
Yeah just need to drop at least one down so this doesn't match: https://caniuse.com/?search=nullish%20coal